x-pack/plugins/ingest_manager/server/services/agent_policy.ts

```diff
@@ -388,9 +402,15 @@ class AgentPolicyService {
       throw new Error('Default output is not setup');
     }
     const defaultOutput = await outputService.get(soClient, defaultOutputId);
+    const settings = await getSettings(soClient);
 
     const fullAgentPolicy: FullAgentPolicy = {
       id: agentPolicy.id,
+      fleet: {
```

Comment:
I think we use that method for standalone mode (you can see the config in the enroll flyout, in the standalone tab), we should probably not have this if `options.standalone === true`